nulib/dc-nextjs
# Case insensitive facet filtering

## What does this do?
In the Filter modal window, facet filtering is now case insensitive thanks for a helper function provided by @mbklein @kdid and @bmquinn 

<img width="697" alt="image" src="https://user-images.githubusercontent.com/3020266/203352068-87f064ba-e927-4cb8-850d-6404c3f6083a.png">


## How to test
1. From the screen shot above, go to a Work, and open the Filter modal
2. Type facet filter values, and notice case insensitivity in results
